docker/sf-gpu.Dockerfile

```diff
@@ -1,3 +1,28 @@
 FROM tensorflow/tensorflow:2.10.1-gpu
-
+#install secretflow
+#you are supposed to add the mirror source of pypi to accelerate installation of SecretFlow and accelerate the building of images
+#if not, the building of images are prone to fail very much 
 RUN pip install -U secretflow
+# remove the cpu version of jax and jaxlib which were installed when run 'pip install -U secretflow'
+RUN pip uninstall -y jax
```

Comment:
This uninstall is unnecessary, you can install cuda and GPU jax prior to secretflow.